lib/metasploit/framework/data_service/remote/http/remote_loot_data_service.rb

```diff
@@ -17,6 +17,7 @@ def loot(opts = {})
       if loot[:data] && !loot[:data].empty?
         local_path = File.join(Msf::Config.loot_directory, File.basename(loot[:path]))
         rv[data.index(loot)].path = process_file(loot[:data], local_path)
+        rv[data.index(loot)].data = decode_loot_data(loot[:data])
```

Comment:
When REMOTE_DB=1 on CI was set the tests failed. It turns out the remote db encodes loot data, but it was never decoded.